api/v1/submissions/PKPSubmissionController.php

```diff
@@ -1706,4 +1758,41 @@ protected function getWriteDisabledErrors(string $schemaName, array $params): ar
 
         return $errors;
     }
+
+    /**
+     * Copy author, files, etc. multilingual fields from old to new changed language
+     */
+    protected function copyMultilingualData(Submission $submission, string $newLocale): void {
+        $oldLocale = $submission->getData('locale');
+        $editProps = fn (Author|SubmissionFile $item, array $props): array => collect($props)
+            ->mapWithKeys(fn (string $p): array => [$p => ($d = $item->getData($p)[$oldLocale] ?? null) ? [$newLocale => $d] : null])
+            ->filter()
+            ->toArray();
+
+        // Submission files
+        $fileProps = [
+            'name',
+        ];
+        Repo::submissionFile()
+            ->getCollector()
+            ->filterBySubmissionIds([$submission->getId()])
+            ->getMany()
+            ->each(fn (SubmissionFile $f) => Repo::submissionFile()->edit($f, $editProps($f, $fileProps)));
+        
+        // Contributor
+        $contributorProps = [
+			'givenName',
```

Comment:
This is the only place I've seen it so far, but this is using tabs when the rest of the codebase here is using spaces. I'd double check all the additions to make sure they're using the same code styling as the specific repo.